go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/opf/openproject-cli
 
-go 1.22
+go 1.22.0
+
+toolchain go1.23.4
```

Comment:
can you elaborate, why setting toolchain and not just go version to 1.23?

related question: any specific reason pinning version to patch version, instead of minor?

Reply:
This is the current toolchain version. just going for the newest one available. `toolchain` is added by `go get -u go@1.23.0` to enable reproducible builds for devs and CI.